docs/source/data-types/blob.md

```diff
@@ -13,11 +13,11 @@ use scylla::IntoTypedRows;
 // We can insert it by reference to not move the whole blob
 let to_insert: Vec<u8> = vec![1, 2, 3, 4, 5];
 session
-    .query("INSERT INTO keyspace.table (a) VALUES(?)", (&to_insert,))
+    .query_unpaged("INSERT INTO keyspace.table (a) VALUES(?)", (&to_insert,))
     .await?;
 
 // Read blobs from the table
-let result = session.query("SELECT a FROM keyspace.table", &[]).await?;
+let result = session.query_unpaged("SELECT a FROM keyspace.table", &[]).await?;
```

Comment:
Also a full scan, just page.

Reply:
There are so many unpaged queries present in our docs... I suggest postponing rewrite of them to a separate follow-up PR.